---
layout: post
title: Research
---

# Peer Reviewed and Published

- **Gerard, D.** (2020). *Data-based RNA-seq simulations by binomial thinning.* BMC Bioinformatics 21(1), 206, p. 1–14. <br> <[doi:10.1186/s12859-020-3450-9](https://doi.org/10.1186/s12859-020-3450-9)> <[bioRxiv:758524](https://doi.org/10.1101/758524)>

- **Gerard, D.**, & Stephens, M. (2020). *Empirical Bayes Shrinkage and False Discovery Rate Estimation, Allowing For Unwanted Variation.* Biostatistics, 21(1), p. 15–32. <br>
<[doi:10.1093/biostatistics/kxy029](https://doi.org/10.1093/biostatistics/kxy029)> <[arXiv:1709.10066](https://arxiv.org/abs/1709.10066)>

- **Gerard, D.** & Ferrão, L. F. V. (2020). *Priors for Genotyping Polyploids.* Bioinformatics 36(6), p. 1795–1800. <br>
<[doi:10.1093/bioinformatics/btz852](https://doi.org/10.1093/bioinformatics/btz852)> <[bioRxiv:751784](https://doi.org/10.1101/751784)>

- **Gerard, D.**, & Stephens, M. (accepted). *Unifying and Generalizing Methods for Removing Unwanted Variation Based on Negative Controls.* Statistica sinica (in press), p. 1–42. <br>
<[doi:10.5705/ss.202018.0345](https://doi.org/10.5705/ss.202018.0345)> <[arXiv:1705.08393](https://arxiv.org/abs/1705.08393)>

- Pollack, J. D., **Gerard, D.**, Makhatadze, G., & Pearl, D. K. (2019). *Evolutionary conservation and structural localizations suggest a physical trace of metabolism’s progressive geochronological emergence.* Journal of Biomolecular Structure and Dynamics, ja, p. 1–20. <br>
<[doi:10.1080/07391102.2019.1679666](https://doi.org/10.1080/07391102.2019.1679666)>

- **Gerard, D.**, Ferrão, L. F. V., Garcia, A. A. F., & Stephens, M. (2018). *Genotyping Polyploids from Messy Sequencing Data.* Genetics, 210(3), p. 789–807. <br>
<[doi:10.1534/genetics.118.301468](https://doi.org/10.1534/genetics.118.301468)> <[bioRxiv:281550](https://doi.org/10.1101/281550)>

- **Gerard, D.**, & Hoff, P. (2017). *Adaptive higher-order spectral estimators.* Electronic Journal of Statistics, 11(2), p. 3703–3737. <br>
<[doi:10.1214/17-EJS1330](http://doi.org/10.1214/17-EJS1330)> <[arXiv:1505.02114](https://arxiv.org/abs/1505.02114)>

- **Gerard, D.**, & Hoff, P. (2016). *A higher-order LQ decomposition for separable covariance models.* Linear Algebra and its Applications, 505, p. 57–84. <br>
<[doi:10.1016/j.laa.2016.04.033](http://doi.org/10.1016/j.laa.2016.04.033)> <[arXiv:1410.1094](https://arxiv.org/abs/1410.1094)>

- **Gerard, D.**, & Hoff, P. (2015). *Equivariant minimax dominators of the MLE in the array normal model.* Journal of Multivariate Analysis, 137, p. 32–49. <br>
<[doi:10.1016/j.jmva.2015.01.020](http://doi.org/10.1016/j.jmva.2015.01.020)> <[arXiv:1408.0424](https://arxiv.org/abs/1408.0424)>

- Pollack, J. D., **Gerard, D.**, & Pearl, D. K. (2013). *Uniquely Localized Intra-Molecular Amino Acid Concentrations at the Glycolytic Enzyme Catalytic/Active Centers of Archaea, Bacteria and Eukaryota are Associated with Their Proposed Temporal Appearances on Earth.* Origins of Life and Evolution of Biospheres, 43(2), p. 161–187. <br>
<[doi:10.1007/s11084-013-9331-8](http://doi.org/10.1007/s11084-013-9331-8)>

- **Gerard, D.**, Gibbs, H. L., & Kubatko, L. (2011). *Estimating hybridization in the presence of coalescence using phylogenetic intraspecific sampling.* BMC evolutionary biology, 11(1), 291, p. 1–12. <br>
<[doi:10.1186/1471-2148-11-291](http://doi.org/10.1186/1471-2148-11-291)>

# Preprints

- **Gerard, D.** (2020). *Pairwise Linkage Disequilibrium Estimation for Polyploids.* bioRxiv, p. 1–19. <br>
<[doi:10.1101/2020.08.03.234476](https://doi.org/10.1101/2020.08.03.234476)>
